asv.conf.json

```diff
@@ -80,7 +82,8 @@
     //     cases: 'linux2', 'win32', 'cygwin', 'darwin'.
     //
     "exclude": [
```

Comment:
can we add here `{"sys_platform": "darwin", "cupy-cuda110": ""}` I was able to setup the environment that way on Darwin

Reply:
Yes; excludes would work if we want to pin each platform to the best available cuda version, and expect our runners to always have installed that version. We could be more flexible and install whatever version matches the output of `nvcc --version`. I'm thinking I should make a setup.py action that performs that test and re-writes the asv config file.

In any case, I should also write some documentation about how to run the benchmarks ( and why ).

Reply:
so on docker/linux this isn't so much an issue, we can have the runner have any version of CUDA we want, up to the version of CUDA available on the host.  On Windows, it's a different story.